src/lib.rs

```diff
@@ -85,8 +78,8 @@ impl From<pyo3::PyErr> for HyperJsonError {
 import_exception!(json, JSONDecodeError);
 
 /// A hyper-fast JSON encoder/decoder written in Rust
-#[py::modinit(_hyperjson)]
-fn init(py: Python, m: &PyModule) -> PyResult<()> {
+#[pymodinit]
+fn hyperjson(py: Python, m: &PyModule) -> PyResult<()> {
```

Comment:
This is also a nice one.

Reply:
Yep, I've just implemented that in pyo3 inspired by #27.